include/miopen/miopen.h

```diff
@@ -6582,6 +6582,40 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+// NLLLoss APIs
+/** @addtogroup nllloss
```

Comment:
Can you check if you can use `LossFunction`? it's already there for CTCLoss.

Reply:
I am working on adding NLLLoss to the LossFunction group, but when I looked at the LossFunction page, it appeared as follows: There is no distinction between CTCLoss and NLLLoss function.
<img width="1547" alt="lossfunctions" src="https://github.com/moreh-dev/MIOpen/assets/54091202/c9209add-ab08-472e-9003-f30469271548">

Reply:
So I decided to add the NLLLoss group to the LossFunction group; created and added the CTCLoss group to the LossFunctions group:
```c++
#ifdef MIOPEN_BETA_API
// NLLLoss APIs
/** @addtogroup LossFunction
 *
 *  @{
 */

/** @addtogroup NLLLoss
*
*  @{
*/

...

/** @} */
// CLOSEOUT NLLLoss DOXYGEN GROUP

/** @} */
// CLOSEOUT LossFunction DOXYGEN GROUP
```
The results are as follows, with each page for each loss function remaining the same. Should I proceed with this?

<img width="644" alt="lossfunctions" src="https://github.com/moreh-dev/MIOpen/assets/54091202/54ed08ce-dc0f-413d-9869-27f6a37986d2">